src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2184,3 +2176,20 @@ Keyword\:=Keyword:
 Keyword\ \"%0\"\ already\ exists=Keyword "%0" already exists
 Remove\ keyword=Remove keyword
 Are\ you\ sure\ you\ want\ to\ remove\ keyword\:\ \"%0\"?=Are you sure you want to remove keyword: "%0"?
+
+Edit\ string\ constants=Edit string constants
+Export\ all\ entries=Export all entries
+Generate\ BibTeX\ keys=Generate BibTeX keys
+Groups\ interface=Groups interface
+ISO=ISO
+MEDLINE=MEDLINE
```

Comment:
MEDLINE and ISO are both acronyms for organizations which are not really translatable, so it makes no sense to have them in here

Reply:
Good point. I will do it.